### PR TITLE
Update diagram tooling to include Visio or PowerPoint

### LINKED ISSUE #921: Update diagram tooling to include Visio or PowerPoint
**Is your feature request related to a problem? Please describe.**

In our last engagement, we used the recommended drawio tooling. We are in the release phase and we have to recreate some of our architecture diagrams to publish in the Azure Architecture Center.

People in the team also think we should not dismiss Microsoft tooling that are at least as good as drawio.

**Describe the solution you'd like**

`docs/design/design-reviews/recipes/preferred-diagram-tooling.md` should include Visio or PowerPoint at least at the same level as drawio.

**Describe alternatives you've considered**

- Choose Visio only. Drawbacks include: no desktop app on the Mac (even though the Web App works great), some customers may not have a license (it's not part of the Office 365 license), there may be legacy diagram habits with drawio
- Choose PowerPoint only. Drawbacks include: the main purpose is not diagramming, Azure icons need to be downloaded separately, there may be legacy diagram habits with drawio
- Choose Visio or Powerpoint only. Drawbacks include: there may be legacy diagram habits with drawio, *.drawio.png and integration with VS code are great features that are unique to drawio.

**Additional context**

N/A

### PR DESCRIPTION
**Pull request by benjguin**
# Pull Request Template

## What are you trying to address

- Update diagram tooling to include Visio or PowerPoint - fixes #921 
- When .drawio format is used, recommend .drawio.png, to complement what was started with #848 

## Description of new changes

- Add Visio and Powerpoint option
- Restructure the page so that the three options are described (Visio, PowerPoint, Drawio)

## For all pull requests

- [x] Link to the issue you are solving (so it gets closed)
- [x] Label the pull request with the appropriate area(s)
- [x] Assign potential reviewers (you may also want to contact them on Teams to ensure timely reviews)
- [ ] Assign the project - Engineering Playbook Backlog
- [x] Assign the pull request to yourself

### CODE REVIEW

File: docs/design/design-reviews/recipes/preferred-diagram-tooling.md [hunk 1]
```diff
@@ -1,15 +1,30 @@
...
-Furthermore, it can be edited within VSCode very easily using the [Draw.io Integration VSCode Extension](https://marketplace.visualstudio.com/items?itemName=hediet.vscode-drawio).
+- Microsoft Visio
+- Microsoft PowerPoint
+- The `.drawio.png` (or `.drawio`) format from [diagrams.net](http://diagrams.net) (formerly [draw.io](http://draw.io))
+
+In all cases, we recommend storing the exported PNG images from these diagrams in the repo along with the source files so they can easily be referenced in documentation and more easily reviewed during PRs. The `.drawio.png` format stores both at once.
+
+## Microsoft Visio
+
+This is the Microsoft diagram tool of choice, the desktop app exists on PC, and there's a great Web app. This is the preferred format of the Azure Architecture Center.
```
Comment (TessFerrandez) [inline docs/design/design-reviews/recipes/preferred-diagram-tooling.md:14]: I think we should hold off on giving a tool of choice, they all have their uses, so I would tone this down a bit, but we can still mention that it is the preferred format for the Azure Architecture Center if they have explicitly expressed that.

**Review by TessFerrandez — APPROVED** [review]
Thank you,

Approving with a suggestion

File: docs/design/design-reviews/recipes/preferred-diagram-tooling.md (hunk 1 above)
Comment (benjguin) [reply docs/design/design-reviews/recipes/preferred-diagram-tooling.md]: Thanks for the heads up @TessFerrandez. Done.

File: docs/design/design-reviews/recipes/preferred-diagram-tooling.md [hunk 2]
```diff
@@ -1,15 +1,30 @@
...
 
 At each stage in the engagement process, diagrams are a key part of the design review.
-The preferred tooling for creating and maintaining diagrams is the `.drawio` format from [diagrams.net](http://diagrams.net) (formerly [draw.io](http://draw.io))
+The preferred tooling for creating and maintaining diagrams is to choose **one** of the following:
 
-The `.drawio` format uses the metadata layer within the PNG file-format to hide SVG vector graphics representation, then renders the .png when saving.
-This clever use of both the meta layer and image layer allows anyone to further edit the PNG file.
-It also renders like a normal PNG in browsers and other viewers, making it easy to transfer and embed.
-Furthermore, it can be edited within VSCode very easily using the [Draw.io Integration VSCode Extension](https://marketplace.visualstudio.com/items?itemName=hediet.vscode-drawio).
+- Microsoft Visio
```
Comment (shiranr) [inline docs/design/design-reviews/recipes/preferred-diagram-tooling.md:6]: NIT: perhaps add links to Visio and Power point?

**Review by shiranr — APPROVED** [review]
LGTM, added a small NIT comment :)

File: docs/design/design-reviews/recipes/preferred-diagram-tooling.md (hunk 2 above)
Comment (benjguin) [reply docs/design/design-reviews/recipes/preferred-diagram-tooling.md:6]: Thanks Shiran. 

I hesitated. Then I thought readers of this content would know where to find Visio or PowerPoint, while they might discover drawio tools.

This does not seem to be a strong requirement on your side either. I'll resolve the conversation as is.

PS: The links that would last quite long would probably have been <https://microsoft.com/visio> and <https://microsoft.com/powerpoint>.

**Pull request merged**